front/assets/js/project_onboarding/new/pages/workflow_setup/project_environment.tsx

```diff
@@ -206,7 +213,17 @@ export const Projectenvironment = () => {
             </div>
             <div className="mt3">
               <div className="flex justify-between items-center">
-                <p className="f6 gray mb0">Next, we&apos;ll define your build steps and workflow.</p>
+                <div className="flex items-center">
+                  <p className="f6 gray mb0 mr1">Next, we&apos;ll define your build steps and workflow. or you can </p>
+                  <a
+                    href="#"
+                    onClick={(e) => { e.preventDefault(); onSkipOnboarding(); }}
```

Comment:
WDYT:

`onClick={onSkipOnboarding}`, and then prevent the event default in the `onSkipOnboarding` callback.